packages/core/src/components/advanced-select/advanced-select.tsx

```diff
@@ -140,49 +116,52 @@ export class AdvancedSelect implements ComponentInterface {
   }
 
   private handleTab(direction: number) {
-    const tabs = this.tabbableElements;
-    const currentIndex = tabs.findIndex((e) => e === this.host.shadowRoot?.activeElement);
+    const elements = tabbable(this.host, { getShadowRoot: true });
```

Comment:
Ah, hier liep ik mee te stoeien. Blijkbaar een makkelijke oplossing voor 😄

Reply:
Ja, `tabbable` is een dependency van `focus-trap` :) We hebben deze gewoon tot onze beschikking.